_locales/da/messages.json

```diff
@@ -1,186 +1 @@
 "searchPlaceholder": {"message": "S\u00f8g...", "description": "The placeholder text in the search input."}, "searchCounting": {"message": "$searchIndex$ af $searchCount$", "description": "The text showing search counting.", "placeholders": {"searchIndex": {"content": "$1", "example": "Search index"}, "searchCount": {"content": "$2", "example": "Search count"}}}, "searchNextButton": {"message": "N\u00e6ste", "description": "The button text to search the next result."}, "searchPreviousButton": {"message": "Forrige", "description": "The button text to search the previous result."}, "errorTitle": {"message": "Fejl", "description": "The error text displayed in the filename title bar."}, "loadingTitle": {"message": "Indl\u00e6ser\u2026", "description": "The text displayed in the filename title bar when file loads."}, "minimizeButton": {"message": "Minimer", "description": "The button text to minimize the app window."}, "maximizeButton": {"message": "Maksim\u00e9r", "description": "The button text to maximize the app window."}, "restoreButton": {"message": "Gendan", "description": "The button text to restore the app window."}, "closeButton": {"message": "Afslut", "description": "The button text to close the app window."}, "yesDialogButton": {"message": "Ja", "description": "The button text which says 'Yes'."}, "noDialogButton": {"message": "Nej", "description": "The button text which says 'No'."}, "cancelDialogButton": {"message": "Annuller", "description": "The button text which says 'Cancel'."}, "saveFilePromptLine1": {"message": "$filename$ er blevet \u00e6ndret.", "description": "The first line of the text which prompts the user to save a file before closing it. This line declares which file has been modified.", "placeholders": {"filename": {"content": "$1", "example": "file.txt"}}}, "saveFilePromptLine2": {"message": "Vil du gemme \u00e6ndringerne, inden du lukker?", "description": "The second line of the text which prompts the user to save a file before closing it. This line asks if the user wants to save the file."}, "okDialogButton": {"message": "OK", "description": "The button text which says 'OK'."}, "closeFileButton": {"message": "Luk fil", "description": "The button text to close a file."}, "screenReaderModeSetting": {"message": "Tilstanden Sk\u00e6rml\u00e6ser", "description": "The label for the checkbox in settings tab that activates screen reader mode, this mode allows for screenreaders to function correctly but disables syntax highlighting, tab control, line wrapping and smart indenting."}, "screenReaderModeHelp": {"message": "Aktiv\u00e9r tilstanden Sk\u00e6rml\u00e6ser i menuen med indstillinger for at f\u00e5 mest muligt ud af din sk\u00e6rml\u00e6ser.", "description": "The message that is read out to a user when they first open the text app with a screen reader, this message should notify users who are using a screen reader about the presence of the screen reader mode setting which optimizes the app for use with a screen reader."}}
```

Comment:
is it a mistake removing the line breaks? here and in all the other files

Reply:
oh yep! my python script was missing a flag, thanks for pointing this out!